app/src/lib/api.ts

```diff
@@ -361,6 +365,48 @@ export interface IAPIRefCheckRuns {
   readonly check_runs: IAPIRefCheckRun[]
 }
 
+interface IAPIWorkflowRuns {
+  readonly total_count: number
+  readonly workflow_runs: IAPIWorkflowRun[]
+}
+// NB. Only partially mapped
+export interface IAPIWorkflowRun {
+  readonly cancel_url: string
+  readonly created_at: string
+  readonly jobs_url: string
+  readonly logs_url: string
+  readonly name: string
+  readonly rerun_url: string
+  readonly check_suite_id: number
+}
+
+interface IAPIWorkflowJobs {
+  readonly total_count: number
+  readonly jobs: IAPIWorkflowJob[]
```

Comment:
Any reason for not using `ReadonlyArray`?
```suggestion
  readonly jobs: ReadonlyArray<IAPIWorkflowJob>
```

Reply:
Original solution I added jobs to existing object - but refactor to not do that. Thanks!